components/dashboard/ActionTableTooltips.tsx

```diff
@@ -23,13 +31,13 @@ const PhasesTooltipList = styled.ul`
   list-style: none;
 `;
 
-const PhasesTooltipListItem = styled.li`
-  color: ${(props) => (props.active === 'true' ? '#333' : '#ccc')};
+const PhasesTooltipListItem = styled.li<{ active: boolean }>`
```

Comment:
I believe custom props (i.e. not passed directly to the DOM element) for styled components should be prefixed with a `$`, otherwise there will be complaints in the browser console: https://styled-components.com/docs/basics#passed-props